100-level-AsyncAPI-Intro/step2.md

```diff
@@ -0,0 +1,329 @@
+As Eve and Chan begin building a communication system using AsyncAPI inside Chan's spaceship control room, Chan asks Eve if there is a standardized way of defining the communication system or if she defines the communication systems differently based on the particular use case.
+
+Eve replies that AsyncAPI has a standardized way of defining communication systems. She explains to Chan that the communication system is defined using a specific structure within an AsyncAPI document. As it is much easier to understand things in action, Eve shows Chan how to structure components in an AsyncAPI document.
+
+## Understanding AsyncAPI components 
+
+- **Info**
+
+Just as the name tag on Chan's spaceship provides essential details like the spaceship's name and model, the `info` section in AsyncAPI serves a similar purpose. It provides crucial information about the API, such as the title, version, and description. The `info` metadata helps developers understand the basic API details they're working with.
+
+```yml
+asyncapi: '3.0.0'
+info:
+  title: Space Communication API
+  version: '1.0.0'
+  description: API for communication between Chan's spaceship and Capuccinova
+```
+
+- **Servers**
+
+Imagine that Chan's spaceship communicates with Eve on Capuccinova through various servers. In AsyncAPI, the `servers` section describes the servers our API will communicate with, including their hosts, protocols, and descriptions.
```

Comment:
AsyncAPI Servers can be either servers **the application of the** API communicates with and also servers the application itself exposes (like a websocket interface, HTTP Rest, etc).